test/helpers/protect-intrinsics-helpers.js

```diff
@@ -28,7 +28,10 @@ function getGlobalProperties() {
       // TODO: Also include the named platform globals
       // This grabs every enumerable property on globalThis.
       // ...Object.keys(globalThis),
-    ].filter((propertyName) => !ignoreList.has(propertyName)),
+    ].filter(
+      (propertyName) =>
+        typeof propertyName === 'string' && !ignoreList.has(propertyName),
```

Comment:
```suggestion
        // dont test symbols because [...]
        typeof propertyName === 'string' && !ignoreList.has(propertyName),
```

Reply:
6a7d51f

Reply:
Fix a4c65ba4fb2bf574cd2abb3804120e2435c0ce9a by properly adding the new symbol to the ignore list to correctly ignore.

The new symbol, introduced in this new lockdown version, is being ignored because it uniquely belong to the globalThis of new compartments but does not belong as a property to the real globalThis (and this lockdown test verifies that the real globalThis is frozen, so this new symbol is irrelevant in this test).

@kumavis

Reply:
In other words, we ignore symbols because the real globalThis doesn't even have any symbol own properties to test lockdown against, so no need for that